src/series.py

```diff
@@ -38,27 +36,35 @@ def sum_series(n, f_num=0, s_num=1):
             second = sum_list[len(sum_list) - 1]
             sum_list.append(first + second)
         return sum_list[n - 1]
-    elif n == 0:
-        raise ValueError
     else:
-        raise ValueError
+        raise TypeError('Input needs to be a positive integer.')
 
 
 if __name__ == '__main__':
-    print(u"This module defines 3 functions that implement different mathematical series.")
+    print(u"""This module defines 3 functions that implement different
+        mathematical series.""")
```

Comment:
This is actually going to show up strangely in the terminal.  Remember, all the spaces in a multi-line string count.

Instead, take advantage of the ability to join strings by abutting them inside a parenthetical statement in Python:

```python
print(u"This module defines 3 functions that implement different "
      u"mathematical series.")
```